suites/pacific/cephmgr/tier-2_verify_throttling_mgr_balancer.yaml

```diff
@@ -0,0 +1,84 @@
+#===============================================================================================
```

Comment:
Does new test suite file required here ? Can't be existing test suite used?

Reply:
Thanks @vamahaja
Updated in existing suite